src/pages/Leaderboard/CardLeaderboard.js

```diff
@@ -22,6 +23,35 @@ class CardLeaderboard extends Component {
         </li>
       ))
 
+    function onHover(value) {
+      const name = value ? document.getElementById('profile-name-' + value) : null
+      const pic = value ? document.getElementById('profile-pic-' + value) : null
+
+      if (pic) {
+        pic.style.border = '2px solid #7EBC89'
+        pic.style.outline = '2px solid #7EBC89'
+        pic.style.outlineOffset = '4px'
+      }
+
+      if (name) {
+        name.style.color = '#7EBC89'
+      }
+    }
+
+    function onLeave(value) {
+      const name = value ? document.getElementById('profile-name-' + value) : null
+      const pic = value ? document.getElementById('profile-pic-' + value) : null
+
+      if (pic) {
+        pic.style.border = null
+        pic.style.outline = null
+        pic.style.outlineOffset = '-8px'
+      }
+      if(name) {
+        name.style.color = '#fff'
+      }
+    }
+
```

Comment:
DOM manipulation is a bit of an anti-pattern in React.  could you try refactoring this code using tailwind? https://tailwindcss.com/docs/hover-focus-and-other-states.  also see this https://stackabuse.com/how-to-style-hover-in-react/

Reply:
I'm looking into group-hover from tailwind. I changed the tailwind.config.js to enable group-hover, but it doesn't seem to work. Any reason for that? Are we using a version of tailwind that doesn't have group-hover?